src/agent/rustjail/src/specconv.rs

```diff
@@ -8,7 +8,7 @@ use oci::Spec;
 #[derive(Serialize, Deserialize, Debug, Default, Clone)]
 pub struct CreateOpts {
     pub cgroup_name: String,
-    pub use_systemd_cgroup: bool,
+    pub use_systemd_cgroup: Option<bool>,
```

Comment:
Out of curiosity, but what's the reason for changing this to Option?

Reply:
that's a great question. It's the first thing i thought of, but i'll see if i can pass the `init_mode` value deeper, so that we have all the info to make the cgroup manager decision.